lib/govuk_ab_testing/acceptance_tests/active_support.rb

```diff
@@ -0,0 +1,36 @@
+module GovukAbTesting
+  module AcceptanceTests
+    class ActiveSupport
+      attr_reader :request, :scope
+
+      def initialize(scope)
+        @request = scope.instance_variable_get(:@request)
+        @scope = scope
+      end
+
+      def set_header(name, value)
+        request.headers[name] = value
+      end
+
+      def vary_header(response)
+        response.headers['Vary']
+      end
+
+      def analytics_meta_tags
+        @analytics_meta_tags ||= scope.css_select("meta[name='govuk:ab-test']")
+      end
+
+      def analytics_meta_tag
+        analytics_meta_tags.first
```

Comment:
Probably now, but we assert there is only one in both test helpers, which is our use case now. I think we shouldn't make it work with more A/B tests for now until we have that need if that makes sense.